src/cljc/rems/common/form.cljc

```diff
@@ -409,3 +409,18 @@
           (is (empty? (validate-visible {:visibility/type :only-if
                                          :visibility/field {:field/id "fld1"}
                                          :visibility/values ["yes"]}))))))))
+
+(defn parse-attachment-ids [string]
+  (mapv parse-int (when-not (empty? string)
+                    (str/split string #","))))
+
+(defn unparse-attachment-ids [ids]
+  (str/join "," ids))
+
+(deftest test-parse-unparse-attachment-ids
```

Comment:
Might be just my brain wiring, but I would prefer if all the parse cases were first and then unparse. :)